include/uuid.h

```diff
@@ -47,6 +47,14 @@ void uuid_to_string(const uuid_t uuid, char *out);
  */
 int uuid_from_string(const char *in, uuid_t uuid);
 
+/**
+ * @brief generate a UUIDv4.
+ *
+ * @details This function computes a UUID in a random way based on esp_fill_random.
```

Comment:
`This function computes a UUID in a random way based on esp_fill_random.` -> `This function computes a random UUID using hardware RNG.`